src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
+		sessions: {},
+		client: client,
+	});
```

Comment:
Actually, this is gross. Please do instead:

```js
this.name = name;
this.sessions = {};
this.client = client;
```

I would imagine the benefit of using [`_.merge`](https://lodash.com/docs#merge) is to be concise, but we even save 2 lines here...
Apart from that, it's useful when you don't know the keys, or when objects are big, or when deep-nesting is an issue... None of which can ever happen here.